conans/client/command.py

```diff
@@ -1388,50 +1388,72 @@ def alias(self, *args):
 
         self._conan.export_alias(args.reference, args.target)
 
-    def link(self, *args):
-        """ Links a conan reference (e.g lib/1.0@conan/stable) with a local folder path.
-
+    def workspace(self, *args):
+        """ command to manage workspaces
         """
-        parser = argparse.ArgumentParser(description=self.link.__doc__,
-                                         prog="conan link")
-        parser.add_argument('target', help='Path to the package folder in the user workspace',
-                            nargs='?',)
-        parser.add_argument('reference', help='Reference to link. e.g.: mylib/1.X@user/channel')
-        parser.add_argument("--remove", action='store_true', default=False,
-                            help='Remove linked reference (target not required)')
-        parser.add_argument("-l", "--layout",
-                            help='Relative or absolute path to a file containing the layout.'
-                            ' Relative paths will be resolved first relative to current dir, '
-                            'then to local cache "layouts" folder')
+        parser = argparse.ArgumentParser(description=self.workspace.__doc__,
+                                         prog="conan workspace")
+        subparsers = parser.add_subparsers(dest='subcommand', help='sub-command help')
+
+        install_parser = subparsers.add_parser('install', help='install this workspace')
+        install_parser.add_argument('path', help='path to workspace file or folder')
```

Comment:
Add something related to the default name of the _workspace file_, was it `conan_ws.yml`?

Reply:
There is no default name now.

Reply:
Ok, then it can no longer be a _path to workspace folder_